packages/tx-construction/test/tx-builder/TxBuilder.test.ts

```diff
@@ -302,6 +353,11 @@ describe('GenericTxBuilder', () => {
       expect(outputBuilder.toTxOut().datumHash).toEqual(datumHash);
     });
 
+    it('can set handle', () => {
+      outputBuilder.handle(handle);
+      expect(outputBuilder.toTxOut().handle).toEqual(handle);
+    });
+
```

Comment:
Missing a test case for this new error
```suggestion
  it('attempting to set handle without handleProvider throws an error');
```